.github/workflows/ci_cron.yml

```diff
@@ -377,11 +347,17 @@ jobs:
         if: ${{ failure() }}
         uses: ./.github/actions/annotate
 
-      - name: Notify slack fail
-        if: ${{ failure() }}
+  notify_slack_fail:
+    name: Notify slack fail
+    needs: [unit_tests, multiverse, infinite_tracing, jruby_multiverse]
+    runs-on: ubuntu-22.04
+    if: always()
+    steps:
+      - uses: technote-space/workflow-conclusion-action@v3
+      - uses: voxmedia/github-action-slack-notify-build@v1
+        if: env.WORKFLOW_CONCLUSION == 'failure'
```

Comment:
The 'technote' action tracks the 'conclusions' of every job and has an overall 'WORKFLOW_CONCLUSION' variable. Now the determining pass/fail is decided by if: env.WORKFLOW_CONCLUSION == 'failure'. And if: always() should ensure the Notify Slack Fail job always runs. 

This change is to limit failure notifications to 1 if the CI fails. VS multiple notifications if multiple jobs have failures.